src/routes/_components/status/Status.html

```diff
@@ -270,6 +270,12 @@
         originalStatus.media_attachments &&
         originalStatus.media_attachments.length
       ),
+      mediaAttachments: ({ originalStatus }) => (
+        originalStatus.media_attachments
+      ),
+      sensitive: ({ originalStatus, $markMediaAsSensitive, $neverMarkMediaAsSensitive }) => (
+        !$neverMarkMediaAsSensitive && ($markMediaAsSensitive || originalStatus.sensitive)
```

Comment:
It's not great that this duplicates code in StatusMediaAttachments, but I wasn't sure how I could get that info without some serious re-plumbing and I'm not quite experienced enough with this code base to pull that off.

Reply:
One way to avoid the duplication would be to pass `sensitive` in to `StatusMediaAttachments`. But It's not a big deal either way.

Reply:
So unfortunately it looks like there's an issue here. If there is a sensitive media, and you click the "show sensitive media" button, then the aria-label doesn't change. It permanently shows the status in "sensitive" mode.

Maybe this wouldn't matter much in practice, but it would matter if you e.g. are scrolling through the timeline, press the `y` button to show the sensitive media, and the aria-label doesn't change at all.

I think in order to get this to work, we would have to plumb from the `StatusMediaAttachments` back to the `Status`. It looks like I already have a store value called `sensitivesShown` that does this. I can look into this.

https://github.com/nolanlawson/pinafore/blob/a775bd9193a8d3b0f24201d09667943210a3e1f4/src/routes/_components/status/StatusMediaAttachments.html#L217-L237

Reply:
This should do it! 470cb15d